### DISCUSSION TITLE
no graphql request and response

### DISCUSSION THREAD

**Discussion by KypMon** [cypress-io/cypress#18120] [Questions and Help]
This seems to be a long lasting issue:

In cypress interface, my application cannot send any graphql request or receive any response. Because it is `fetch` type. 

here is the network status in cypress:

[![enter image description here][1]][1]

But in normal browser, I actually have several graphql requests, like here: 

[![enter image description here][2]][2]

I know there are already quite several discussions and workarounds, such as using an polyfill to solve this problem such as below:

+ https://gist.github.com/yagudaev/2ad1ef4a21a2d1cfe0e7d96afc7170bc
+ https://stackoverflow.com/questions/60827832/cypress-does-not-intercept-graphql-api-calls

but unfortunately, they are not working in my case. 

Appreciate to the help of any kinds.

p.s.: I am using cypress 8.3.0, React as the front-end, and using apollo client and apollo server for all graphql stuff.

  [1]: https://i.stack.imgur.com/hQB8z.png
  [2]: https://i.stack.imgur.com/KPmaN.png

**Comment by san-slysz** [comment]
Fetch is suppose to work in v7+ if you use cy.intercept() (not cy.server or ...).
Works on my side.

So i guess you're just not using [cy.intercept()](https://docs.cypress.io/api/commands/intercept#Syntax)?
